src/utils/records.js

```diff
@@ -1,6 +1,20 @@
 import { addressUtils } from '@0xproject/utils'
 import { validateContent } from './contents'
+
+/**
+ * @typedef { "address" | "content" | "oldcontent" } recordType
```

Comment:
I think you mixed up [type](https://github.com/ensdomains/ens-app/blob/25a55adecadeaf3571b4095a0b2682058f7643d9/src/utils/records.js#L12) and contentType (oldcontent| contenthash | null).

Reply:
Good spot. Should `recordType` have just the following: ` * @typedef { "address" | "content" } recordType`?

Reply:
should be added now